src/app/clients/clients-routing.module.ts

```diff
@@ -149,6 +151,14 @@ const routes: Routes = [
         ]
       }
     ]
+  },
+  {
+    path: 'clients/:clientId/paycharge/:chargeId',
```

Comment:
this should be a nested child route of clients -> :clientId -> ... see rest of clients routing module to see how routes are nested

Reply:
Yuh but it will look like the following:
![Screenshot from 2020-05-12 17-25-00](https://user-images.githubusercontent.com/43112139/81685374-a34c6a00-9475-11ea-9d34-a94ad0e9a942.png)

Reply:
@abhaychawla I guess this is due to router outlet in clients view, how should we route here?

Reply:
In my opinion your approach is fine

Reply:
@karantakalkar  Yeah router outlet was the problem.

Reply:
@BLasan nest the routes here itself to get breadcrumbs right.

Reply:
Is nesting okay? But the outcome will be similar to the above screenshot I attached

Reply:
by nesting here i mean make client id a child of route here itself not in other instance of route present in module